packages/phone-number-privacy/src/database/wrappers/account.ts

```diff
@@ -19,7 +19,23 @@ export async function getPerformedQueryCount(account: string): Promise<number> {
 }
 
 /*
- * Increments query count in database.
+ * Returns whether account has already performed matchmaking
+ */
+export async function getDidMatchmaking(account: string): Promise<boolean> {
+  try {
+    const didMatchmaking = await getDatabase()(ACCOUNTS_TABLE)
+      .where(ACCOUNTS_COLUMNS.address, account)
+      .select(ACCOUNTS_COLUMNS.didMatchmaking)
+      .first()
+    return didMatchmaking !== undefined && didMatchmaking[ACCOUNTS_COLUMNS.didMatchmaking] !== null
```

Comment:
Nit: Unless you need to check for `undefined` or `null` *specifically*, the more common thing in JS would be just:
`didMatchmaking && didMatchmaking[ACCOUNTS_COLUMNS.didMatchmaking]` which will coalesce types for you